### PR TITLE
Fix translations

### PR DESCRIPTION
**Pull request by nop33**

### CODE REVIEW

File: locales/de-DE/App.json [hunk 1]
```diff
@@ -243,5 +238,7 @@
   "Label": "Etikett",
   "Last used": "Zuletzt benutzt",
   "Transactions": "Vorgänge",
-  "ALPH amount": "ALPH-Betrag"
+  "ALPH amount": "ALPH-Betrag",
+  "Set lock time": "Set lock time",
+  "Tweak gas settings": "Tweak gas settings"
```
Comment (nop33) [inline locales/de-DE/App.json:243]: @Iix I could use your help with these 2 new sentences! You can just comment here and I'll add it. 🙏 Much obliged!

File: locales/fr-FR/App.json [hunk 2]
```diff
@@ -247,5 +242,7 @@
   "Label": "Label",
   "Last used": "Dernier usage",
   "Transactions": "Transactions",
-  "ALPH amount": "Montant en ALPH"
+  "ALPH amount": "Montant en ALPH",
+  "Set lock time": "Set lock time",
+  "Tweak gas settings": "Tweak gas settings"
```
Comment (nop33) [inline locales/fr-FR/App.json:247]: @mvaivre 🙏

File: locales/vi-VN/App.json [hunk 3]
```diff
@@ -243,5 +238,7 @@
   "Label": "Nhãn",
   "Last used": "Lần cuối cập nhật",
   "Transactions": "Giao dịch",
-  "ALPH amount": "Số lượng ALPH"
+  "ALPH amount": "Số lượng ALPH",
+  "Set lock time": "Set lock time",
+  "Tweak gas settings": "Tweak gas settings"
```
Comment (nop33) [inline locales/vi-VN/App.json:243]: @nit27 I could use your help with these 2 new sentences! You can just comment here and I'll add it. 🙏 Much obliged!

Comment (nit27) [reply locales/vi-VN/App.json:243]: "Set lock time": "Đặt thời gian khoá",
 "Tweak gas settings": "Tuỳ chỉnh cài đặt gas"
✈️

File: locales/de-DE/App.json (hunk 1 above)
Comment (Iix) [reply locales/de-DE/App.json:243]: "Set lock time": "Sperrdauer einstellen",
"Tweak gas settings": "Gas Einstellungen optimieren"

File: locales/vi-VN/App.json (hunk 3 above)
Comment (nit27) [reply locales/vi-VN/App.json:243]: "Set lock time": "Đặt thời gian khoá",
  "Tweak gas settings": "Tuỳ chỉnh cài đặt gas"

File: locales/fr-FR/App.json (hunk 2 above)
Comment (Iix) [reply locales/fr-FR/App.json:247]: "Set lock time": "Régler le temps de verrouillage",
 "Tweak gas settings": "Ajuster les paramètres du gaz"

**Comment by nop33** [comment]
@Iix @nit27 you guys rock 🤘 Thanks a lot

**Pull request merged**